### ISSUE TITLE
Accessibility audit: Error for Multiple form labels

### ISSUE THREAD

**Issue by zoeabryant** [redbadger/website-honestly#587]
**Description of the issue:**
http://wave.webaim.org/ accessibility audit raises one error:

```
**Errors**
Multiple form labels

**What It Means**
A form control has more than one label associated with it.

**Why It Matters**
A form control should have at most one associated label element. If more than one label element is associated to the control, assistive technology may not read the appropriate label.

**How to Fix It**
Ensure that at most one label element is associated to the form control. If multiple form labels are necessary, use aria-labelledby.
```
The multiple form labels are for the checkbox that opens and closes the navigation on mobile. We should try and hide the label that is hidden with `display: none`.

**Browser used:** n/a
**OS used:** n/a
**Device used:** n/a

**Steps to reproduce:**

1. Go to http://wave.webaim.org/
2. Put in the URL to the main blog page.
3. Currently it is one of 43 errors, but the rest of those errors are fixed in this PR: https://github.com/redbadger/blog-squarespace-template/pull/89

**Actual result:**
`Errors (43)`

**Expected result:**
`Errors (0)`

**Comment by dpiatek** [comment]
Fixed in https://github.com/redbadger/website-honestly/pull/739
